client/my-sites/checkout/composite-checkout/wpcom/types/wpcom-store-state.ts

```diff
@@ -1,3 +1,18 @@
+/**
+ * Internal dependencies
+ */
+import {
+	DomainContactDetails,
+	PossiblyCompleteDomainContactDetails,
+	DomainContactDetailsErrors,
+	CaDomainContactExtraDetails, // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
What's the reason for these being imported if they're not being used?

Reply:
#40482 is getting huge, partly because it does a bunch of additional refactoring that isn't strictly related to the problem of adding the ccTLD fields. With this and #41055 (and some others to come) I'm trying to pull out some of the incidental complexity. These will be used in a later patch.

Reply:
Could they be imported in that patch? It just seems like adding imports to a patch is unlikely to cause issues since you'd have to change these lines in that patch anyway to remove the `eslint-disable-line` comments. I'm happy to be talked into liking this, though, if you can explain more.